floris/tools/uncertainty_interface.py

```diff
@@ -1,337 +1,174 @@
-            -   **pdf_cutoff** (*float*): A float containing the cumulative
-                distribution function value at which the tails of the
-                PMFs are truncated.
-            Defaults to None. Initializes to {'std_wd': 4.95, 'pmf_res': 1.0,
-            'pdf_cutoff': 0.995}.
-        unc_pmfs (dictionary, optional): A dictionary containing optional
-            probability mass functions describing the distribution of wind
-            direction deviations. Contains the following key-value pairs:
-            -   **wd_unc** (*np.array*): Wind direction deviations from the
-                original wind direction.
-            -   **wd_unc_pmf** (*np.array*): Probability of each wind
-                direction deviation in **wd_unc** occuring.
-            Defaults to None, in which case default PMFs are calculated
-            using values provided in **unc_options**.
-        fix_yaw_in_relative_frame (bool, optional): When set to True, the
-            relative yaw angle of all turbines is fixed and always has the
-            nominal value (e.g., 0 deg) when evaluating uncertainty in the
-            wind direction. Evaluating  wind direction uncertainty like this
-            will essentially come down to a Gaussian smoothing of FLORIS
-            solutions over the wind directions. This calculation can therefore
-            be really fast, since it does not require additional calculations
-            compared to a non-uncertainty FLORIS evaluation.
-            When fix_yaw_in_relative_frame=False, the yaw angles are fixed in
-            the absolute (compass) reference frame, meaning that for each
-            probablistic wind direction evaluation, our probablistic (relative)
-            yaw angle evaluated goes into the opposite direction. For example,
-            a probablistic wind direction 3 deg above the nominal value means
-            that we evaluate it with a relative yaw angle that is 3 deg below
-            its nominal value. This requires additional computations compared
-            to a non- uncertainty evaluation.
-            Typically, fix_yaw_in_relative_frame=True is used when comparing
-            FLORIS to historical data, in which a single measurement usually
-            represents a 10-minute average, and thus is often a mix of various
-            true wind directions. The inherent assumption then is that the turbine
-            perfectly tracks the wind direction changes within those 10 minutes.
-            Then, fix_yaw_in_relative_frame=False is typically used for robust
-            yaw angle optimization, in which we take into account that the turbine
-            often does not perfectly know the true wind direction, and that a
-            turbine often does not perfectly achieve its desired yaw angle offset.
-            Defaults to fix_yaw_in_relative_frame=False.
+        wd_resolution (float, optional): The resolution of wind direction, in degrees.
```

Comment:
Maybe not here, but in general we should pick either "wind_direction" or "wd" for wind direction named arguments.

Reply:
And same for the other related input arguments

Reply:
Yeah, I would be happy either way, with maybe a slight preference toward wd_ for succinctness in this case